accesshandler/pkg/providers/aws/flask/setup/3.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Finalizing Your Deployment
+configFields:
+---
+
+# Finalizing Your Deployment
+
+You can now test the connection!
+
+Afterwards you will be able to:
+- Create a Flask access rule
+- Create an access request for the Flask deployment
+- Assume into the ECS cluster
+
+To finalise the setup you will need to install [granted-flask](https://pypi.org/project/granted-flask/)
+-  More on this here [docs.commonfate.io/granted-approvals/providers/flask](https://docs.commonfate.io/granted-approvals/providers/flask)
+
+Once you have updated your Flask deployment with the correct webhook URL you can test your connection with the `test` command
+Run the following to test your configuration is correctly setup:
+```
+aws ecs execute-command --cluster {CLUSTER_ARN} --task {TASK_ID} --container DefaultContainer --interactive --command 'flask test'
```

Comment:
I like the test command but we shouldn't make this a flask command unless we're overriding functionality.

Can we replace this with `granted test` instead?